# Height of xy plane marker
svm_thre: -0.1

# Grid resolution for position [m]
pos_resolution: 0.05

# Grid resolution for rotation [rad]
rot_resolution: 0.05
